src/fiatExchanges/FiatExchangeAmount.test.tsx

```diff
@@ -103,41 +104,20 @@ describe('FiatExchangeAmount cashIn', () => {
     jest.clearAllMocks()
     storeWithUSD.clearActions()
     storeWithPHP.clearActions()
-  })
-
-  it('renders correctly with USD as app currency', () => {
-    const mockScreenProps = getMockStackScreenProps(Screens.FiatExchangeAmount, {
-      currency: CiCoCurrency.cUSD,
-      flow: CICOFlow.CashIn,
-      network: Network.Celo,
-    })
-    const tree = render(
-      <Provider store={storeWithUSD}>
-        <FiatExchangeAmount {...mockScreenProps} />
-      </Provider>
-    )
-    expect(tree).toMatchSnapshot()
-  })
-
-  it('renders correctly with EUR as app currency', () => {
-    const mockScreenProps = getMockStackScreenProps(Screens.FiatExchangeAmount, {
-      currency: CiCoCurrency.cUSD,
-      flow: CICOFlow.CashIn,
-      network: Network.Celo,
+    jest.mocked(getFeatureGate).mockImplementation((gate) => {
+      return gate === StatsigFeatureGates.SHOW_NATIVE_TOKENS ? true : false
```

Comment:
wow, this is one of the larger facepalms i've had in a while

Reply:
perhaps something like this could work instead...

```suggestion
      return (gate === StatsigFeatureGates.SHOW_NATIVE_TOKENS ? (true ? true : false) : (false ? true : false))
```